xdcs-frontend/src/styles/basic/colors.less

```diff
@@ -0,0 +1,4 @@
+.colors('basic') {
```

Comment:
a z kim to uzgodniłeś, kolego?

Reply:
wiele osób się ze mną nie zgadza, na szczęście ja się z sobą zgadzam